test/commands/build-nodetest.js

```diff
@@ -64,7 +63,7 @@ describe('custom configuration', function() {
 
   before(function() {
     app = new AddonTestApp();
-    return app.create('custom').then(prepareApp);
+    return app.create('custom');
```

Comment:
@tchak Why remove the `prepareApp` call here? I think this would cause the test to fail due to the `ember-cli-fastboot` dependency missing.

Reply:
@tchak perhaps try adding this back and see?